tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts

```diff
@@ -109,37 +113,58 @@ suite(`Check if recommended extensions installed for ${samples}`, async function
         test(`Let extensions complete installation`, async function (): Promise<void> {
             Logger.info(`Time for extensions installation TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT=${TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT}`);
             await driverHelper.wait(TimeoutConstants.TS_COMMON_PLUGIN_TEST_TIMEOUT);
-            await browserTabsUtil.refreshPage();
-            await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
         });
 
         test(`Check if extensions is installed and enabled`, async function (): Promise<void> {
+            this.retries(10);
```

Comment:
does this mean the test will be re-executed 10 times until success or eventual failure?

Reply:
yes, test can fail because extension is still installing

Reply:
I would suggest to leave no more 3 retries, to avoid too long test execution log.

Reply:
We have 5 retries as default and ti is not enough. If you will look to this log you will see that 'java' extension test was passed on 5th time only.
```
    ✔ Let extensions complete installation (30029ms)
      • Check if {"publisher":"redhat","name":"java"} are installed.
      • Check if {"publisher":"redhat","name":"java"} are installed.
      • Check if {"publisher":"redhat","name":"java"} are installed.
      • Check if {"publisher":"redhat","name":"java"} are installed.
      • Check if {"publisher":"redhat","name":"java"} are installed.
      • Check if {"publisher":"vscjava","name":"vscode-java-debug"} are installed.
```

Reply:
ACK, thanks for explanation

Reply:
Got it. Thanks.